zmq-multiplex.slide

```diff
@@ -0,0 +1,136 @@
+ZMQ multiplex between golang coroutine and python thread
+10 June 2017
+
+Rick Mak
+Oursky
+rickmak@oursky.com
+@rickmak
+CC81327D
+
+* What we want to do
+
+- Enable building of scalable software with less learning curve
+- Provide foundation for developer to focus business requirement and UI
+- Save developer from deployment hassle
+
+* Skygear-server
+
+- Design for web development
+- Written in golang
+- Provide basic function like Auth, Record CRUD, PubSub via core
+- Provide plugin for extending function, fulfilling business requirement.
+
+* ZMQ
+
+- High performance messaging queue
+- No message broker
```

Comment:
People may ask: there is so many messaging queue solutions out there (RabbitMQ?) why use this ZMQ thing? Answer: it is a low-level library so that we can customize message routing (something like that, not sure if it is correct).